Makefile

```diff
@@ -48,6 +48,14 @@ SRC_DIRS += $(CUNIT_SRC_DIRS)/Framework
 #SRC_DIRS += $(CUNIT_SRC_DIRS)/Win
 #SRC_DIRS += $(CUNIT_SRC_DIRS)/Test
 
+# Enable libyaml Requirements
+LIBFYAML_DIR += $(UT_DIR)/framework/libfyaml-master
+PKG_CONFIG_PATH += $(LIBFYAML_DIR)
+CFLAGS += $(shell pkg-config --cflags libfyaml)
+LDFLAGS += $(shell pkg-config --libs libfyaml)
+SRC_DIR_FYAML += $(UT_DIR)/tests/kvp
```

Comment:
tests should only be built by the test makefile, not by this one -> Needs to move to the `tests/Makefile`

This one is the production file.

Reply:
done